code/workspaces/infrastructure-api/src/kubernetes/deploymentGenerator.js

```diff
@@ -146,9 +150,10 @@ function createSparkDriverHeadlessService(notebookName) {
 }
 
 function createPySparkConfigMap(notebookName, projectKey) {
+  const img = returnImage('spark');
```

Comment:
Should the `'spark'` and `'dask'` be capitalized to match the way they are in the config map? I think they get capitalized somewhere else in the code but it might be nice to have the connection directly where it is used.